docs/04-contributing/development_environment.md

```diff
@@ -0,0 +1,239 @@
+There are multiple ways to get started.
```

Comment:
You still need to start with a heading 1. Like we had before.

Also a it's not a development environment, a development environment is workspace for developers to make changes without breaking anything in a live environment. Some people even use development environment to refer to their IDE. I think it's better to just keep it "development server". Which is really what we try to setup here.

Reply:
:smile:  this has always been a problem in every company I've been in. Does dev env mean the developer's machine or the deployed environment used by devs. There's even https://devenv.sh/ which described itself as "Fast, Declarative, Reproducible and Composable Developer Environments using Nix" which refers to the local machine.

I don't mind changing the headers, but what should they become?

```md
# Development server

## Docker

## Host machine
````

Where "host machine" is non-docker environment?

Reply:
Development environment is literately the virtual place where you indeed develop code (hence the name). Just like integrated development environment, refers to the piece of software that help you write that code. 

So devenv is important within a company, but we aren't providing any development environment like maybe VDIs (Virtual Desktop Infrastructure) to your contributors. 

So I still believe it's a development server (it's also not a staging server), but really a local development server we try to aim to setup here. And then the software engineer can choose between either a docker setup or just "bare metal" is what we called it thus far.  "Host machine" might be an alternative for "bare metal". 
Changing the filename also means that it will break search engine indexed results (like this URL will not work anymore: https://docs.joinmbin.org/contributing/development_server).

I think the top-level heading might still then be called "Development Server" indeed, then "Docker". And well, we could go for Host machine, maybe that is confusing for most people (since Docker also runs on your host machine), we also could call it "Bare metal"? We could even considering splitting the documentation. Look how we did it for "Installation": https://docs.joinmbin.org/admin/installation/

Reply:
I finally split it in two (docker + bare metal). It looks like a good pattern to follow especially due to how long the files can become.